misc/bazel/registry/modules/rules_nodejs/6.2.0-codeql.1/patches/module_dot_bazel_version.patch

```diff
@@ -0,0 +1,14 @@
+===================================================================
+--- a/MODULE.bazel
++++ b/MODULE.bazel
+@@ -1,9 +1,9 @@
+ "bzlmod declaration for bazelbuild/rules_nodejs"
+ 
+ module(
+     name = "rules_nodejs",
+-    version = "0.0.0",
++    version = "6.2.0",
```

Comment:
if anyone asks, this doesn't need to updated. Actually this whole patch is unneeded, the `MODULE.bazel` file present in the downloaded module is not used, the one that is used is the copy in the registry.

Reply:
but I left the patch as it's copied from the bazel central registry... but I could remove it

Reply:
I went ahead and did that